### PR TITLE
Created an associative table called EmployeeTraining

### PR DESCRIPTION
**Pull request by kayrahbear**
# Illegal Llamas Pull Request

## Status
This pull request is ready to be reviewed by my peers.

## Description
Added a new model titled EmployeeTraining. This associative table will map the many to many relationship between the Employee table and the Training Program table. 

### Related Tickets 
***

[Link to relevant ticket](https://github.com/illegal-llamas/Bangazon-LLC/issues/52)

### Impacted Areas of the Application
***
Files that this PR modifies or affects in some way. 
```
    models.py

```

### Deploy Notes
***
Because this table is an associative table, it is not exposed to the API and cannot be testing using the standard migrate and runserver method. 
```
    git fetch --all
    git checkout kb-creatingEmployeeTrainingJoinTable
    inspect the models.py file (found within Banazon-app/bangazon/api/) for the EmployeeTraining model
```

### CODE REVIEW

**Comment by jnelsontn** [comment]
This looks fine. 💯

File: bangazon-app/bangazon/api/models.py [hunk 1]
```diff
@@ -190,6 +192,16 @@ class CustomerSupport(models.Model):
...
+    """
+    author: Kayla Brewer
+    purpose: This model describes and maps the relationship between the Employee table and the Training Program table.
+    properties: 
+                employeeID: foreign key constraint to Employee table
+                TrainingProgramID: foreign key constraint to Training Program table 
+    
+    """
+    Employee = models.ForeignKey(Employee)
+    TrainingPrograms = models.ForeignKey(TrainingProgram)
```
Comment (hfrankst) [inline bangazon-app/bangazon/api/models.py]: 'TrainingPrograms' should be singular

**Comment by thenicknash** [comment]
Change variable names to lower case and then it looks fantastic!

**Comment by hfrankst** [comment]
Looks good :shipit:

**Pull request merged**